Assets/Scripts/Models/World.cs

```diff
@@ -26,7 +26,7 @@ public class World : IXmlSerializable
     public readonly string GameVersion = "Someone_will_come_up_with_a_proper_naming_scheme_later";
     public List<Character> characters;
     public List<Furniture> furnitures;
-    public List<Room> rooms;
+    public RoomManager roomManager;
```

Comment:
for this I followed the structure of the one below, the `InventoryManager`. But I do think an accessor would be better.